drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -3494,7 +3502,7 @@ VkRenderPass RenderingDeviceVulkan::_render_pass_create(const Vector<AttachmentF
 			} else {
 				ERR_FAIL_INDEX_V_MSG(attachment, p_attachments.size(), VK_NULL_HANDLE, "Invalid framebuffer format attachment(" + itos(attachment) + "), in pass (" + itos(i) + "), resolve attachment (" + itos(j) + ").");
 				ERR_FAIL_COND_V_MSG(pass->color_attachments[j] == FramebufferPass::ATTACHMENT_UNUSED, VK_NULL_HANDLE, "Invalid framebuffer format attachment(" + itos(attachment) + "), in pass (" + itos(i) + "), resolve attachment (" + itos(j) + "), the respective color attachment is marked as unused.");
-				ERR_FAIL_COND_V_MSG(!(p_attachments[attachment].usage_flags & TEXTURE_USAGE_COLOR_ATTACHMENT_BIT), VK_NULL_HANDLE, "Invalid framebuffer format attachment(" + itos(attachment) + "), in pass (" + itos(i) + "), it's marked as depth, but it's not usable as resolve attachment.");
+				ERR_FAIL_COND_V_MSG(!(p_attachments[attachment].usage_flags & TEXTURE_USAGE_RESOLVE_ATTACHMENT_BIT), VK_NULL_HANDLE, "Invalid framebuffer format attachment(" + itos(attachment) + "), in pass (" + itos(i) + "), it isn't marked as a resolve texture.");
```

Comment:
RESOLVE_ATTACHMENT_BIT should probably be removed from RenderingDevice instead since there is no need for it, COLOR_ATTACHMENT is the correct one.

Reply:
It was that originally and it didn't work, Vulkan validation layers also complained about it. This is why I couldn't get MSAA to work, it was just throwing errors that I was using an image not marked as a resolve attachment.

Reply:
Ok so it was just for INPUT_ATTACHMENTS that it was important, not for resolve, so I'll change that.